### ISSUE TITLE
hide scroll at page refresh / open new page

### ISSUE THREAD

**Issue by esseifforte** [inuyaksa/jquery.nicescroll#784]
Hello,
maybe this question has been asked over and over again, but I honestly did not find an appropriate solution to the problem.
As the title suggests, every time I reload the page or jump to another page, I would not want the scroll to appear immediately but to start seeing only when scrolling.
thank you.

**Comment by Jewinn** [comment]
Definitely, it's appearing on page load, please suggest how to disable that!
